config/config.yaml

```diff
@@ -165,4 +165,9 @@ default:
             - stream:
                 name: command_stream
                 output: 3075
-                command_subscriber: True
\ No newline at end of file
+                command_subscriber: True
+
+
+    extensions:
+        ait.core.cmd.Cmd: ait.core.newCmd.NewCmd
+        ait.core.table.FSWTabDict: ait.core.newTable.NewFSWTabDict
```

Comment:
These are testing codes?

Reply:
Yes that is for testing.  I took it out.